beetmoverscript/src/beetmoverscript/gcloud.py

```diff
@@ -208,6 +208,27 @@ async def push_to_releases_gcs(context):
                 )
             else:
                 log.debug("Excluding partner repack {}".format(blob_path))
+        # TODO: Remove this block when it's time to archive all the released .debs, we'd like to start by shipping beta and devedition.
+        elif "b" in context.task["payload"]["version"]:  # if we are shipping a beta build, we actually want to archive the .deb
```

Comment:
Nit: Let's use mozilla-version as a source of truth to detect beta.

```py
from mozilla_version.gecko import FirefoxVersion
#... 

version = FirefoxVersion.parse(context.task["payload"]["version"])
elif version.is_beta
```

Reply:
I thought it would be over-kill since we are going to remove this in January, but I could install and uninstall it.

Reply:
Good news: no need to install it, it's already used in beetmover 😃 

https://github.com/mozilla-releng/scriptworker-scripts/blob/b2d06c796ba9668db610e78c993e3391de9fa28f/beetmoverscript/requirements/base.in#L4